scripts/src/cowidev/xm/etl.py

```diff
@@ -102,6 +99,32 @@ def run(self):
         self.load(df, PATHS.DATA_XM_MAIN_FILE, date_accessed)
 
 
+class XMortalityEconomistETL:
+    def extract(self):
+        cat = catalog.RemoteCatalog(channels=["garden"])
+        t = cat.find_latest(namespace="excess_mortality", dataset="excess_mortality_economist", table="excess_mortality_economist")
+        date_accessed = max(s.date_accessed for s in t.metadata.dataset.sources)
+        return pd.DataFrame(t), date_accessed
+
+    def transform(self, df: pd.DataFrame) -> pd.DataFrame:
+        return df.sort_values(["country", "date"])
+
+    def load(self, df: pd.DataFrame, output_path: str, date_accessed: str) -> None:
+        df_current = pd.read_csv(output_path)
+        if not df.equals(df_current):
+            # Export data
+            __import__('ipdb').set_trace()
```

Comment:
what is this magic?

Reply:
That's a forgotten breakpoint... which is why we're doing reviews!